semcore/utils/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.14.0] - 2023-11-28
+
+### Added
+
+- New `--intergalactic-border-critical-pattern` token for non-color marking all types of inputs with `invalid` state.
```

Comment:
I recommend to change the note to smth like added `...` token for marking controls invalid state